bfver

```diff
@@ -108,6 +108,21 @@ print_bfb_file_vers () {
     # Keep UEFI version and use it to determine BSP version as well
     UEFI_VER="$(strings -el "$BFB_PATH" | grep "BlueField" | cut -d':' -f 2)"
 
+    UEFI_IMAGE="$BFB_PATH"
```

Comment:
Maybe a comment here about the purpose of this code? Something like "The UEFI image may be compressed. If it is, decompress before searching for version strings."

Reply:
done